project_timesheet_time_control/views/project_task_view.xml

```diff
@@ -93,6 +93,9 @@
                     </group>
                 </form>
             </xpath>
+            <xpath expr="//field[@name='timesheet_ids']/tree" position="attributes">
+                <attribute name="default_order">date_time</attribute>
```

Comment:
Why adding this?

Reply:
The "timesheet_ids" field in the Tasks view form has default_order = "date" specified, so i write that to order timesheets by date_time field insted.